testData/sample_tests/test/simple_outline.txt

```diff
@@ -2,11 +2,13 @@
   "kind": "DART_ELEMENT",
   "offset": 0,
   "length": 878,
+  "codeOffset": 0,
+  "codeLength": 878,
   "dartElement": {
     "kind": "COMPILATION_UNIT",
     "name": "<unit>",
     "location": {
-      "file": "/usr/local/google/home/djshuckerow/Code/github.com/DaveShuckerow/flutter-intellij/testData/sample_tests/test/simple_test.dart",
+      "file": "/usr/local/google/home/jacobr/git/flutter-intellij/1/flutter-intellij/testData/sample_tests/test/simple_test.dart",
```

Comment:
Not necessarily for this PR, but I wonder if we shouldn't normalize these files somehow (to sanitize paths).

Reply:
that would be nice but lets leave it for a cleanup pr